### DISCUSSION TITLE
lesson-audio/cherokee-language-lessons-1---audio-exercises/

### DISCUSSION THREAD

**Discussion by giscus** [CherokeeLanguage/CherokeeLessons.com#13] [General]
# lesson-audio/cherokee-language-lessons-1---audio-exercises/

https://www.cherokeelessons.com/lesson-audio/cherokee-language-lessons-1---audio-exercises/

**Comment by BooksandTea** [comment]
I have the second edition of Cherokee Language Lessons 1, does that still work for this?

**Reply by michael-conrad** [reply]
Yes, the two editions are close enough that this audio will work for the 2nd edition.

**Reply by michael-conrad** [reply]
Updated audio uploaded: 2022-08-23 13:28:15

**Comment by CharlieMcVicker** [comment]
Hi! I am so thankful for your work in making these materials publicly available! I have a question about some of the lesson content and was wondering the best way for me to provide feedback, but I'll just put it in a comment for now!

There is a frequent example sentence "ᏐᏈᎵ ᎠᏂᎪᏩᏘᎭ" which I thought meant "They see a horse" but it is translated as "They see horses" - Am I just not hearing the verb correctly?

I would also love to learn more about your other open source work, as I have experience in NLP/DS/DeepLearning and would love to contribute to creating tools for revitalizing the language. Let me know the best way to be in touch!

**Reply by michael-conrad** [reply]
I'm glad you like what we've produced.

In regards to ᏐᏈᎵ ᎠᏂᎪᏩᏘᎭ, to say "they see horses" it would be ᏐᏈᎵ ᏓᏂᎪᏩᏘᎭ.

The prefix ᎠᏂ- means: "They(subj)+It/Him/Her(obj)"
The combined prefixes Ꮥ- ᎠᏂ- ⇒ ᏓᏂ- means: "They(subj)+them(obj)"

We have an organization on github [https://github.com/CherokeeLanguage/](https://github.com/CherokeeLanguage/) where we make available various repos for things like the IMS-Toucan TTS, various book sources (LyX), websites, audio lesson generator, etc.

**Reply by michael-conrad** [reply]
If that is in the audio, then it needs fixing. I'll check.

**Reply by michael-conrad** [reply]
Yes, it's a typo in the audio transcript file. I'll upload new audio "soon". It takes a bit to generate.

FYI: The replacement Cherokee audio will be spoken slower which will change up the timing and duration of the audio files.

**Reply by michael-conrad** [reply]
https://github.com/CherokeeLanguage/audio-lessons-generator-python/issues/1

**Reply by michael-conrad** [reply]
@CharlieMcVicker

If interested, I can provide you a link to a Telegram chat group we have set up for Cherokee Language Tech discussions, comments, etc.

**Reply by CharlieMcVicker** [reply]
I'd love to be connected - what is the best way to exchange that link? Do you have a public email I could reach out to?

**Reply by michael-conrad** [reply]
Updated audio uploaded: 2022-08-23 13:28:15

**Comment by michael-conrad** [comment]
I've added the option to download all the MP3s in a single zip:

[cll1-v3-audio.zip](https://www.cherokeelessons.com/lesson-audio/cherokee-language-lessons-1---audio-exercises/cll1-v3-audio.zip)
